source/i18n/scripts.ttl

```diff
@@ -1,104 +1,108 @@
-# Start here: <https://en.wikipedia.org/wiki/ISO_15924>
+# For a list of available alphabets/scripts, start here:
+#   <https://en.wikipedia.org/wiki/ISO_15924>
+# When you add a new script or need to update all labels; get labels for all by calling:
+#   $ python3 scripts/construct.py source/remote/construct-matching-wd-labels.rq THIS_FILE
+# then add the resulting :prefLabel data here.
```

Comment:
Maybe add a reminder that there should always be a `@sv`-tagged label

Reply:
Good point; added in 64c600f6316a95f0dbad9422dd8bb7eb64d160e6